api/db/seeds/data/common/tooling/campaign-tooling.js

```diff
@@ -458,6 +466,42 @@ function _buildCampaign({
 
 let emailIndex = 0;
 
+function _createAnonymousParticipation(databaseBuilder, organizationId, campaignId, createdAt) {
+  const createdDate = dayjs(createdAt)
+    .add(_.random(0, _numberOfDaysBetweenNowAndCreationDate(createdAt)), 'days')
+    .toDate();
+
+  const organizationLearnerId = databaseBuilder.factory.buildOrganizationLearner({
+    firstName: ``,
+    lastName: ``,
+    sex: 'M',
+    birthdate: '2000-01-01',
+    birthCity: null,
+    birthCityCode: '75115',
+    birthCountryCode: '100',
+    birthProvinceCode: null,
+    isDisabled: false,
+    createdAt: new Date(),
+    updatedAt: new Date(),
+    organizationId,
+    userId: null,
+  }).id;
+
+  databaseBuilder.factory.buildCampaignParticipation({
+    campaignId,
+    userId: null,
```

Comment:
**To do :** Pour les anonymes, la participation est bien relié à un user avec la propriété `isAnonymous`